modules/run-vault/run-vault

```diff
@@ -228,6 +265,19 @@ function run {
       --skip-vault-config)
         skip_vault_config="true"
         ;;
+      --enable-s3-backend)
+        enable_s3_backend="true"
+        ;;
+      --s3-bucket)
+        assert_not_empty "$key" "$2"
+        s3_bucket="$2"
+        shift
+        ;;
+      --s3-bucket-region)
+        assert_not_empty "$key" "$2"
+        s3_bucket_region="$2"
+        shift
+        ;;
```

Comment:
Around line 245 there are some `assert_not_empty` checks. If `enable_s3_backend` is `true`, you should add some `assert_not_empty` checks for the s3 bucket name & region configs.